widgets/src/list/rasterizer.rs

```diff
@@ -59,7 +59,7 @@ impl<R: Renderer> WidgetRasterizer<R> for ListTransmogrifier {
                 child_bounds =
                     child_bounds.translate(content_area.location + Vector::from_x(offset_amount));
 
-                if let Some(indicator) = indicators.next().unwrap() {
+                if let Some(indicator) = indicators.next().flatten() {
```

Comment:
I looked into this and unfortunately I don't think this is the right fix. The iterator is supposed to produce an exact number of indicators for the children -- the fact it isn't is a bug. I have a theory -- once you changed this, did you find that you had forgotten to register a component? If your app is displaying all of the widgets currently, can you try removing this and see if you still have the panic?

If it is working now, that's great, it confirms my suspicions. If it's still failing... I'd love to see a bit more on Discord!

For now this is innocuous to have in your local branch -- I'd just rather try to track down what's causing this iterator to not produce enough values.